library/src/main/java/com/novoda/downloadmanager/MigrationStatus.java

```diff
@@ -1,7 +1,26 @@
 package com.novoda.downloadmanager;
 
-interface MigrationStatus {
+public interface MigrationStatus {
 
-    String message();
+    enum Status {
+
+        EXTRACTING,
+        MIGRATING_FILES,
+        DELETING_V1_DATABASE,
+        COMPLETE;
+
+        public String toRawValue() {
+            return this.name();
+        }
+
+    }
+
+    int numberOfBatches();
+
+    int totalNumberOfBatches();
```

Comment:
is this the absolute number of batches that needs to be migrated?

Reply:
Yes 😄

Reply:
e2b956c1a199d8766e391b136a63c9b265ff03ba